### PR TITLE
EG-64 Change django-oscar pin to < 1.6 and run make upgrade

### PR DESCRIPTION
**Pull request by crice100**
Update django-oscar to the latest 1.5.x version, which is 1.5.4, since 1.6 contains backwards incompatible changes that I'd like to avoid for now.

Note that this seems to downgrade _django-tables2_ from 1.21.2 to 1.16.0.

Oscar release notes:
https://django-oscar.readthedocs.io/en/latest/releases/v1.5.2.html
https://django-oscar.readthedocs.io/en/latest/releases/v1.5.3.html
https://django-oscar.readthedocs.io/en/latest/releases/v1.5.4.html

### CODE REVIEW

File: requirements/base.in [hunk 1]
```diff
@@ -13,7 +13,7 @@ django-crispy-forms
 django_extensions==1.9.9
 django-filter==1.0.4
 django-libsass==0.5
-django-oscar==1.5.1
+django-oscar==1.5.4
```
Comment (cpennington) [inline requirements/base.in]: I think we should unpin this and let `pip-compile` force the pin during `make upgrade`.

Comment (crice100) [reply requirements/base.in]: I updated the description. 1.5.4 is the latest release that avoids backwards incompatible changes (should have made that clear from the beginning, sorry).

Comment (cpennington) [reply requirements/base.in]: Sorry, can you make sure that this explanation is in the code, and not just in the PR description? In the commit message would be ok-er, but as a comment is probably best.

Comment (crice100) [reply requirements/base.in]: Sure.

**Review by cpennington — APPROVED** [review]

**Comment by crice100** [comment]
@nedbat figured out that this is what downgraded _django-tables2_ : https://github.com/django-oscar/django-oscar/commit/f9746f00a3c48dd02175178c2c7909d5bd942e87

**Pull request merged**